library/system/io.lean

```diff
@@ -266,17 +266,7 @@ This primitive is used to invoke external tools (e.g., SAT and SMT solvers) from
 IMPORTANT: this primitive can be used to implement `unsafe_perform_io {α : Type} : io α → option α`
 or `unsafe_perform_io {α : Type} [inhabited α] : io α → α`. This can be accomplished by executing
 the resulting tactic using an empty `tactic_state` (we have `tactic_state.mk_empty`).
```

Comment:
This remark doesn't work since we don't have `tactic_state.mk_empty`, right?  If that's correct we should remove it.

Reply:
yep